packages/PlanLimitsUI/src/pages/Limits/index.tsx

```diff
@@ -25,21 +24,24 @@ export const defaultViewLocation = {
   longitude: 175.35,
   zoom: 8,
 };
+export const defaultPath = `/limits/${createLocationString(
+  defaultViewLocation
+)}`;
 
 export type WaterTakeFilter = 'Surface' | 'Ground' | 'Combined';
 
 export const loader: LoaderFunction = ({ params, request }) => {
+  if (!params.location) return redirect(defaultPath);
```

Comment:
I'm a fan of this early-return code style for reducing nested conditionals, and improving readability by reducing the context required as you scan down a function.

Reply:
Yes, I get sometimes torn between the guideline of "one exit path for functions" and how simply something like what you've done there makes it. 
Seems like this "early-return code style" if a good place to ignore the guideline ... actually you made me do some googling http://wiki.c2.com/?GuardClause